docs/projects/hello-world/_posts/2018-11-17-red.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Hello World in Red
+layout: default
+last-modified: 2018-11-17
+featured-image:
+tags: [red, hello-world]
+authors:
+  - the_renegade_coder
+---
+Anyway, let’s get right to our implementation of Hello World in Red:
```

Comment:
I updated the template recently. You can check out #731 to see what I'm talking about. Basically, I want to put in `## How to Implement the Solution` here with a short intro paragraph like:

> In this article, we'll demonstrate how to implement Hello World in Red.
 
Let me know if that makes sense.

Reply:
Makes sense. so all articles will start with that title and intro, and below those just paste whatever was in the original post (for the ones we are moving).

Reply:
Just the project articles. I did that because some articles will need to have multiple subsections, and it didn't make sense to have some like:

```
# Title

### Implementation Subsection 1

### Implementation Subsection 2

## How to Run the Solution
```

Clearly, we need a heading in between the title and the implementation.

The language articles don't have this problem.

Reply:
got you